src/main/java/hudson/plugins/repo/TagAction.java

```diff
@@ -23,68 +23,50 @@
  */
 package hudson.plugins.repo;
 
-import hudson.model.Run;
-import hudson.scm.AbstractScmTagAction;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 import org.kohsuke.stapler.export.ExportedBean;
 
+import hudson.model.Run;
+import jenkins.model.RunAction2;
+
 /**
- * A Tag Action allows a user to tag a build. Repo doesn't support a solid tag
- * method, so right now we just display the static manifest information needed
- * to recreate the exact state of the repository when the build was ran.
+ * TagAction class. This class contains code to migrate from the old TagAction
+ * to a ManifestAction, as the TagAction (1.10.7) conflicts with the builtin tag code.
  */
 @ExportedBean(defaultVisibility = 999)
-public class TagAction extends AbstractScmTagAction {
+public class TagAction implements RunAction2 {
```

Comment:
Try to keep the original binary signature, and change the restriction so noone can use it when they upgrade their dependency.
```suggestion
@Restricted(NoExternalUse.class)
public class TagAction implements AbstractScmTagAction {
```

Reply:
Sounds good to me. I've done this now.